web/app/habit/stake/components/stakeContent.tsx

```diff
@@ -49,28 +45,20 @@ export default function StakeChallenge() {
     [challenge?.public, challenge?.accessCode, inputAccessCode, joined],
   );
 
-  const { data: capabilities } = useCapabilities();
-  const currentChainSupportBatchTx =
-    capabilities?.[EXPECTED_CHAIN.id.toString() as unknown as keyof typeof capabilities]
-      ?.atomicBatch.supported;
-
   const { data: testTokenBalance } = useBalance({
     address: smartWallet,
     token: testTokenContract.address,
+    query: {
+      enabled: !!smartWallet && !!challenge,
+      refetchInterval: (data: any) => (challenge && data.value && (Number(data.value.toString()) > Number(challenge.stake.toString())) ? false : 2000),
```

Comment:
Will this cause errors? Like after stake, the data won't be updated?

Reply:
updated to 10000 : 2000